models/collection.go

```diff
@@ -39,6 +39,24 @@ type CollectionHistory struct {
 	Dropped bool
 }
 
+func (c *Collection) GetPKField() (FieldSchema, bool) {
```

Comment:
May BirdWatcher import Milvus as a lib then able to use these utils?

Reply:
Then birdwatcher will be too large